v0.1/core/annotation.schema.json

```diff
@@ -15,49 +22,41 @@
-      "description": "anatomicalEntity that is defined with annotation."
-    },
-    "criteriaType": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
-      "description": "How was this annotation determined."
-    },
     "criteria": {
       "oneOf": [{
-        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json/PROTOCOL"
+        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json/PROTOCOL"
       }, {
-        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v1.0/definitions.json/PROTOCOL_EXECUTION"
+        "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json/PROTOCOL_EXECUTION"
       }],    
       "description": "criteria"
     },
-    "inspiredBy": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/openMINDS/master/v3.0/definitions.json#definitions/FILE_INSTANCE",
-      "description": "inspiredBy"
+    "criteriaType": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM",
+      "description": "How was this annotation determined."
     },
-    "annotationFile": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/ANNOTATION_FILE",
-      "description": "annotationFile"
+    "definingTerm": {
+      "$ref": "https://schema.hbp.eu/sands/core/anatomicalEntity.schema.json",
+      "description": "Anatomical entity that provides the term for the here specified annotation."
     },
-    "nativeCoordinateSpace": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/COORDINATE_SPACE",
-      "description": "The native coordinate space of this annotation."
+    "inspiredBy": {
+      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/IMAGE",
+      "description": "Link to (source) image(s) that inspired the definition of this annotation."
     },
-    "navigationRules": {
-      "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/NAVIGATION_RULES",
-      "description": "navigationRules"
-    }
     "laterality": {
       "type": "array",
-      "description": "The side of the body or bilateral organ that this anatomical entity relates to.",
+      "description": "The side of the body or bilateral organ that this annotation is defined in.",
       "items": {
         "$ref": "https://raw.githubusercontent.com/HumanBrainProject/SANDS/master/v0.1/definitions.json#definitions/CONTROLLED_TERM"
       },
       "maxItems": 2
+    },
+    "relatedAtlasTerm": {
+      "$ref": "https://schema.hbp.eu/sands/core/anatomicalEntity.schema.json",
```

Comment:
Same here. Needs to point to definitions file.

Reply:
see above